.github/workflows/build.yml

```diff
@@ -53,7 +61,7 @@ jobs:
         uses: turtlesec-no/get-ninja@main
 
       - name: Make sure MSVC is found when Ninja generator is in use
-        if: ${{ runner.os == 'Windows' }}
+        if: runner.os == 'Windows'
```

Comment:
I know this looks nicer, but someone might be tempted to write: `if : runner.os != 'Windows' ` which is not valid yaml syntax, so for consistency it's better to keep the escaping tags

Reply:
Aha! I missed that. I'll fix those up.